packages/nimble-components/src/tree-view/tests/tree.spec.ts

```diff
@@ -81,25 +93,70 @@ describe('TreeView', () => {
         );
     });
 
+    it('should include a role of `tree`', () => {
+        expect(model.treeView.getAttribute('role')).toBe('tree');
+    });
+
     it('root1 should have "group-selected" attribute set after initialization', () => {
         expect(model.root1.hasAttribute('group-selected')).toBe(true);
         expect(model.root2.hasAttribute('group-selected')).toBe(false);
     });
 
     it('when leaf item is selected, only root parent tree item has "group-selected" attribute', async () => {
+        let promise = waitForGroupSelectedUpdate();
         await clickElement(model.leaf3);
+        await promise;
         expect(model.root2.hasAttribute('group-selected')).toBe(true);
         expect(model.subRoot2.hasAttribute('group-selected')).toBe(false);
         expect(model.root1.hasAttribute('group-selected')).toBe(false);
         expect(model.leaf3.hasAttribute('group-selected')).toBe(false);
 
+        promise = waitForGroupSelectedUpdate();
         await clickElement(model.leaf1);
+        await promise;
         expect(model.root2.hasAttribute('group-selected')).toBe(false);
         expect(model.subRoot2.hasAttribute('group-selected')).toBe(false);
         expect(model.root1.hasAttribute('group-selected')).toBe(true);
         expect(model.leaf3.hasAttribute('group-selected')).toBe(false);
     });
 
+    it('when leaf item is deselected, root parent tree loses "group-selected" attribute', async () => {
+        await clickElement(model.leaf3);
+        model.leaf3.selected = false;
+        await waitForGroupSelectedUpdate();
```

Comment:
I'm not sure why a single call to `waitForUpdatesAsync()` isn't sufficient for this.

Reply:
Fixed